invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/InvitationResultItem.js

```diff
@@ -20,6 +20,23 @@ import { RequestActionController } from "@js/invenio_requests";
 const formattedTime = (expires_at) =>
   DateTime.fromISO(expires_at).setLocale(i18next.language).toRelative();
 
+export const dropdownRoleOptionsSerializer = (roles) =>
+  roles.map((role) => ({
+    key: role.name,
+    text: role.title,
+    value: role.name,
+    content: (
+      <List>
```

Comment:
Question: the `List` cmp should be only one cmp that wraps the items, and not inside each element.